### PR TITLE
Feature/74 refactor business partner code

### PR DESCRIPTION
**Pull request by gregdeane**
#74

### CODE REVIEW

File: src/app/components/business-partner/business-partner-menu/business-partner-menu.controller.js [hunk 1]
```diff
@@ -9,18 +7,20 @@ class BusinessPartnerMenuController {
...
 
   $onInit() {
     this.$ngRedux.connect(
       BusinessPartnerMenuController.mapStateToProps,
-      Actions
+      this.businessPartnerActions
     )(this);
 
     // TODO Fetch last accessed business partners from BE
+    // TODO YOU SHOULDN'T NEED TO FETCH FROM BE. THIS DATA SHOULD ALREADY BE IN THE STORE
```
Comment (dami-gg) [inline src/app/components/business-partner/business-partner-menu/business-partner-menu.controller.js]: According to this then it's a new backend call to add to the startup. I was thinking to do it only when opening the menu, which won't necessarily happen if there's one BP in the cookie

File: src/app/components/business-partner/business-partner-menu/business-partner-menu.component.spec.js [hunk 2]
```diff
@@ -73,7 +90,7 @@ describe('Business Partner Menu Component', () => {
           businessPartnerMenuShown: false
         }
       },
-      bp: {}
+      businessPartner: {}
```
Comment (dami-gg) [inline src/app/components/business-partner/business-partner-menu/business-partner-menu.component.spec.js]: Even if it's a test we shall better stick to this kind of notation for future reviewers, thanks for doing that :)

**Comment by dami-gg** [comment]
:+1:

File: src/app/components/business-partner/business-partner-menu/business-partner-menu.controller.js (hunk 1 above)
Comment (gregdeane) [inline src/app/components/business-partner/business-partner-menu/business-partner-menu.controller.js]: Ah, okay. I'll remove the comment.

**Comment by gregdeane** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**